.chloggen/add-operator-cli-args.yaml

```diff
@@ -0,0 +1,12 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: breaking
+# The name of the component, or a single word describing the area of concern, (e.g. agent, clusterReceiver, gateway, operator, chart, other)
+component: operator
+# A brief description of the change. Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Refactored operator to use cli args instead of feature gates to control instrumentation availability
```

Comment:
Can you please clarify why this has to be done? Now, instead of flipping each feature gate, users have to override the whole `operator.manager.extraArgs` list

Reply:
Oh, I see we are just adopting the upstream change...